src/tiles/expandable-tile.component.ts

```diff
@@ -22,11 +22,11 @@ export interface ExpandableTileTranslations {
 			type="button"
 			(click)="onClick()">
 			<div [attr.aria-label]="(expanded ? collapse : expand).subject | async" class="bx--tile__chevron">
-				<svg *ngIf="!expanded" width="12" height="7" viewBox="0 0 12 7" role="img">
+				<svg *ngIf="!expanded" width="12" height="7" viewBox="0 0 12 7" title="{{expand.subject | async}}" role="img">
 					<title>{{expand.subject | async}}</title>
```

Comment:
Do we need the `<title>` elements within the SVGs as well? Seems redundant...

Reply:
It was an issue picked by the IBM Accessibility Checker, it might be that some screen readers don't pick up the <title> element inside the svg. The solution given by the plugin for elements with a "img" role with non visible labels is to provide a title attribute within the them.

Reply:
Gotcha!